templates/master.yaml.tmpl

```diff
@@ -3096,6 +3097,7 @@ systemd:
       /hyperkube kubelet \
       --config=/etc/kubernetes/config/kubelet.yaml \
       --node-ip=${DEFAULT_IPV4} \
+      --container-runtime-endpoint=/var/run/dockershim/dockershim.sock \
```

Comment:
I understood that default path for this would be `/var/run/dockershim.sock` - is there some reason to put it under own directory?

I think that e.g. use of `kubesquash` would be easier when you don't need to know and specify the location of this socket: https://github.com/solo-io/kubesquash/blob/master/cmd/kubesquash/main.go#L44 WDYT?

Reply:
Completely agree with that However I've already done the try but kubelet doesn't start and get this error:

```
`Jan 23 22:17:21 ip-10-0-5-235 sh[16181]: F0123 22:17:21.188498   16208 server.go:261] failed to run Kubelet: failed to create kubelet: failed to listen on "unix:///var/run/dockershim.sock": failed to unlink socket file "/var/run/dockershim.sock": is a directory`
```

this because dockershim.sock does not exist on the host, docker thought you intention is to mount it as a directory
so it created an empty directory there instead

That is the reason for this choice

Reply:
Oh, wow! Makes sense. Would be perhaps nice to check later if there's a way to get this fixed, but for now I think this is a good workaround.